index.js

```diff
@@ -112,38 +113,59 @@ async function handleIncreaseScores (req, res, redis, signerAddresses) {
     body.scores.splice(index, 1)
   }
 
-  const timestamp = new Date()
-  const tx = redis.multi()
-  for (let i = 0; i < body.participants.length; i++) {
-    const address = body.participants[i]
-    const score = body.scores[i]
-    const scheduledRewards = (BigInt(score) * roundReward) / maxScore
-    tx.hincrby('rewards', address, scheduledRewards)
-    addLogJSON(tx, {
-      timestamp,
-      address,
-      score,
-      scheduledRewardsDelta: String(scheduledRewards)
+  if (body.participants.length === 0) {
+    return json(res, {})
+  }
+
+  const scheduledRewardsDelta = body.scores.map(score => {
```

Comment:
I find it a bit surprising that a variable with singular name ("...Delta") is storing an array of values.

`scheduledRewardsDeltas` would be a better name (IMO).

Reply:
+1 https://github.com/filecoin-station/spark-rewards/commit/db9a5aa586a3cfa4a8104f6dcabddc06ca95a88b